k8s/helmfile/env/local/argo-cd-base.values.yaml.gotmpl

```diff
@@ -1 +1,18 @@
-# local uses production configuration so this is an empty file
+configs:
+  params:
+    server.insecure: "true"
```

Comment:
question: why set this true but not for the other environments? In any case trying to access it after port forwarding to localhost:8080 will probably result in a TLS error right?

Reply:
just for convenience. The cert changes rapidly for local and hopefully never on the other envs, so you can add exceptions for staging and prod

Reply:
ah cool! i hadn't thought of how quickly they refresh!

Reply:
I should be more clear: I think it only happens once per initial argo setup, but that can be quite often the case locally (especially currently)